raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -79,12 +84,12 @@
     public static final String QUORUM_REQUEST_TIMEOUT_MS_CONFIG = QUORUM_PREFIX +
         CommonClientConfigs.REQUEST_TIMEOUT_MS_CONFIG;
     public static final String QUORUM_REQUEST_TIMEOUT_MS_DOC = CommonClientConfigs.REQUEST_TIMEOUT_MS_DOC;
-    public static final int DEFAULT_QUORUM_REQUEST_TIMEOUT_MS = 20_000;
+    public static final int DEFAULT_QUORUM_REQUEST_TIMEOUT_MS = 4_000;
```

Comment:
Does it make sense for this to be lower than the fetch timeout? After the fetch timeout, we effectively give up on pending requests anyway.

Reply:
Let's set them both to 3 seconds